partner/README.md

```diff
@@ -10,59 +12,60 @@ Install all requirements via [pip-tools][pip-tools]:
 pip-sync requirements.txt dev-requirements.txt
 ```
 
-[pip-tools]: https://pypi.org/project/pip-tools/
-
-## Running the service
+## Code checks and tests
 
-Build the Docker image:
+With requirements installed run the code checks and test via [tox][tox]:
 
 ```text
-docker build -t partner .
+tox
 ```
 
-Run the container:
+See the tox configuration in the `tox.ini` for the list of environments this
+will run.
+
+## Running the service
+
+You can run the service using `docker compose` from the root directory:
 
 ```text
-docker run --rm -p 8000:8000 partner
+docker compose run -p 5000:5000 partner
 ```
 
 ## Tiles API
 
 Example request:
 
 ```text
-curl -X 'GET' \
-  'http://0.0.0.0:8000/tilesp?partner=demofeed&sub1=123456789&sub2=placement1&country-code=US&region-code=NY&form-factor=desktop&os-family=macOS&v=1.0&out=json&results=2' \
-  -H 'accept: application/json'
+curl \
+  -X 'GET' \
+  -H 'accept: application/json' \
+  'http://0.0.0.0:5000/tilesp?partner=demofeed&sub1=123456789&sub2=placement1&country-code=US&region-code=NY&form-factor=desktop&os-family=macos&v=1.0&out=json&results=2'
 ```
 
 Example response body:
 
 ```json
-[
-  {
-    "id": 12345,
-    "name": "tile 12345",
-    "click_url": "example click_url",
-    "image_url": "example image_url",
-    "impression_url": "example impression_url",
-    "advertiser_url": "example advertiser_url"
-  },
-  {
-    "id": 56789,
-    "name": "tile 56789",
-    "click_url": "example click_url",
-    "image_url": "example image_url",
-    "impression_url": "example impression_url",
-    "advertiser_url": "example advertiser_url"
-  }
-]
+{
+  "tiles": [
+    {
+      "id": 12346,
+      "name": "Example",
+      "click_url": "https://example.com/desktop_macos?version=16.0.0",
+      "image_url": "https://example.com/desktop_macos01.jpg",
+      "impression_url": "https://example.com/desktop_macos?id=0001",
+      "advertiser_url": "https://www.example.com/desktop_macos"
+    },
+    {
+      "id": 56790,
+      "name": "Example",
```

Comment:
might want to rename this here and elsewhere to distinguish it from the other, e.g. "Example 2"

Reply:
I created #17 to update this across all components once we've merged this pull request series.